nri-config-generator/main.go

```diff
@@ -139,3 +144,10 @@ func loadConfigTemplate() (*template.Template, error) {
 	}
 	return t, nil
 }
+
+func prometheusExportersBinPath(name string) string {
+	if runtime.GOOS == "windows" {
+		return filepath.Join("C:\\Program Files\\Prometheus-exporters\\bin", fmt.Sprintf("%s.exe", name))
```

Comment:
woud make sense also to move the win path to a constant next to the `nixExporterBinPath` ?

Reply:
totally